galley/pkg/config/analysis/diag/level.go

```diff
@@ -38,3 +42,24 @@ var (
 	// Error level is for error messages
 	Error = Level{0, "Error"}
 )
+
+func GetAllLevels() []Level {
+	return []Level{Info, Warning, Error}
+}
+
+func GetAllLevelStrings() []string {
+	levels := GetAllLevels()
+	var s []string
+	for _, l := range levels {
+		s = append(s, l.name)
+	}
+	return s
+}
+
+func GetUppercaseStringToLevelMap() map[string]Level {
+	m := make(map[string]Level)
+	for _, l := range GetAllLevels() {
+		m[strings.ToUpper(l.name)] = l
+	}
+	return m
+}
```

Comment:
Public functions need doc comments, or the linter will get grumpy.

Reply:
It didn't seem to get grumpy? I'll add them anyways because I think they're useful, but do we need to file an issue against the linter?